logging/log.go

```diff
@@ -1,15 +1,23 @@
 package logging
 
 import (
-	"os"
+	"io"
+	"io/ioutil"
```

Comment:
ioutil was deprecated in go1.16. Can u please update imports to use io package?

Reply:
sure, kinder